.github/workflows/pr-main-awesome.yml

```diff
@@ -0,0 +1,56 @@
+name: pr-main-awesome
+
+on:
+  pull_request:
```

Comment:
You should not monitor pr for publishing data. Instead, you should listen to code push. Please refer to main.yml.

Reply:
I have changed the pull_request to push in two yml file, please check it.